inject/src/main/java/io/avaje/inject/Component.java

```diff
@@ -67,9 +67,10 @@
   @Target({TYPE, PACKAGE, MODULE})
   @interface Import {
 
-    /**
-     * Types to generate DI classes for.
-     */
+    /** Types to generate DI classes for. */
     Class<?>[] value();
+
+    /** Whether the imported types should have the prototype scope. */
+    boolean prototype() default false;
```

Comment:
This would apply to all classes in this import.  That leads me to think that we likely need an additional ImportList annotation and a `@Repeatable(ImportList.class)` so that we can do something like:

```java
@Component.Import(value = SomeComponent.class)
@Component.Import(value = MyProtoComponent2.class, prototype=true)
```

Reply:
Or alternatively a different annotation like `ImportPrototype`